#robot's geometric parameters:

#Diameter of std. mecanum wheel
wheelDiameter: 0.254
#Length of the mobile robot from wheel center to wheel center
robotLength: 0.5
#Width from wheel center to wheel center
robotWidth: 0.548
#stddev vel x
devX: 0.2
#stddev vel y
devY: 0.2
#stddev vel z
devZ: 0.2
#stddev vel roll
devRoll: 0.2
#stddev vel pitch
devPitch: 0.2
#stddev vel x yaw
devYaw: 0.2